sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitService.java

```diff
@@ -35,17 +35,17 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(PhpUnitService.class);
 
-  private final PhpUnitOverallCoverageResultParser overallCoverageParser;
-  private final PhpUnitItCoverageResultParser itCoverageParser;
-  private final PhpUnitCoverageResultParser coverageParser;
-  private final PhpUnitResultParser parser;
+  private final PhpUnitOverallCoverageResultImporter overallCoverageParser;
+  private final PhpUnitItCoverageResultImporter itCoverageParser;
+  private final PhpUnitCoverageResultImporter coverageParser;
+  private final PhpUnitTestResultImporter parser;
 
   private final FileSystem fileSystem;
 
-  public PhpUnitService(FileSystem fileSystem, PhpUnitResultParser parser,
-                       PhpUnitCoverageResultParser coverageParser,
-                       PhpUnitItCoverageResultParser itCoverageParser,
-                       PhpUnitOverallCoverageResultParser overallCoverageParser) {
+  public PhpUnitService(FileSystem fileSystem, PhpUnitTestResultImporter parser,
```

Comment:
@inverno I think we can consider dropping this class. All the logic could be moved to `PhpUnitImporter`